internal/env/config.go

```diff
@@ -50,6 +51,11 @@ type Config struct {
 	AtomicUpgrade      bool     `split_words:"true"`                 // Pass --atomic to `helm upgrade`
 	CleanupOnFail      bool     `envconfig:"cleanup_failed_upgrade"` // Pass --cleanup-on-fail to `helm upgrade`
 	LintStrictly       bool     `split_words:"true"`                 // Pass --strict to `helm lint`
+	ConvertV2Releases  bool     `split_words:"true" default:"true"`  // Whether or not to use 2to3 convert to migrate Releases from v2 to v3
```

Comment:
This would be a major default behavior change from the upstream. Would default false be better?

Reply:
I know a made an opinionated decision here. If we go with false then people would need to explicitly add this parameter in their pipelines. If we want the experience of simply changing `drone-helm:v2` with `drone-helm:v3` and will work, then this should be `true` by default.

For general consumption I agree that the best value is set it to false by default.

Reply:
I removed all default values